packit_service/worker/tasks.py

```diff
@@ -74,3 +113,226 @@ def babysit_copr_build(self, build_id: int):
+    )
+    return get_handlers_task_results(handler.run_job(), event)
+
+
+@celery_app.task(name="task.run_push_koji_build_handler")
+def run_push_koji_build_handler(event: dict, package_config: dict, job_config: dict):
+    handler = PushGithubKojiBuildHandler(
+        package_config=PackageConfigSchema().load_config(package_config),
+        job_config=JobConfigSchema().load_config(job_config),
+        data=EventData.from_event_dict(event),
+    )
+    return get_handlers_task_results(handler.run_job(), event)
+
+
+@celery_app.task(name="task.run_distgit_commit_handler")
+def run_distgit_commit_handler(event: dict, package_config: dict, job_config: dict):
+    handler = NewDistGitCommitHandler(
+        package_config=PackageConfigSchema().load_config(package_config),
+        job_config=JobConfigSchema().load_config(job_config),
+        data=EventData.from_event_dict(event),
+    )
+    return get_handlers_task_results(handler.run_job(), event)
+
+
+@celery_app.task(name="task.run_pagure_pr_comment_copr_build_handler")
+def run_pagure_pr_comment_copr_build_handler(
+    event: dict, package_config: dict, job_config: dict
+):
+    handler = PagurePullRequestCommentCoprBuildHandler(
+        package_config=PackageConfigSchema().load_config(package_config),
+        job_config=JobConfigSchema().load_config(job_config),
+        data=EventData.from_event_dict(event),
+    )
+    return get_handlers_task_results(handler.run_job(), event)
+
+
+@celery_app.task(name="task.run_pagure_pr_label_handler")
+def run_pagure_pr_label_handler(event: dict, package_config: dict, job_config: dict):
+    labels = event.get("labels")
+    action = PullRequestLabelAction(event.get("action"))
+    base_repo_owner = event.get("base_repo_owner")
+    base_repo_namespace = event.get("base_repo_namespace")
+    base_repo_name = event.get("base_repo_name")
+
+    handler = PagurePullRequestLabelHandler(
+        package_config=PackageConfigSchema().load_config(package_config),
+        job_config=JobConfigSchema().load_config(job_config),
+        data=EventData.from_event_dict(event),
+        labels=labels,
+        action=action,
+        base_repo_owner=base_repo_owner,
+        base_repo_name=base_repo_name,
+        base_repo_namespace=base_repo_namespace,
+    )
+    return get_handlers_task_results(handler.run_job(), event)
+
+
+@celery_app.task(name="task.run_koji_build_report_handler")
+def run_koji_build_report_handler(event: dict, package_config: dict, job_config: dict):
+    handler = KojiBuildReportHandler(
+        package_config=PackageConfigSchema().load_config(package_config),
+        job_config=JobConfigSchema().load_config(job_config),
+        data=EventData.from_event_dict(event),
+        koji_event=KojiBuildEvent.from_event_dict(event),
+    )
+    return get_handlers_task_results(handler.run_job(), event)
+
+
+def get_handlers_task_results(results: dict, event: dict):
+    return {"job": results, "event": event}
```

Comment:
Why is the event dict included here?

Reply:
I can remove it, but then the results would look like:
`{'copr_build-20200701101150167006': {'success': True, 'details': {}}}`
which doesn't tell much in my opinion.

Reply:
No need to remove, but maybe add a comment to explain that it is included to provide more context in the result.